### PR TITLE
Merge updates for v1.0.0-0.1.0 release

### PR DESCRIPTION
**Pull request by firestarman**
This PR is to merge the latest updates from internal branch `rel-master` for v1.0.0-0.1.0 release.

### CODE REVIEW

File: jvm-packages/pom.xml [hunk 1]
```diff
@@ -24,9 +24,9 @@
         </developer>
     </developers>
     <scm>
-        <connection>scm:git:git:/github.com/NVIDIA/spark-xgboost.git</connection>
-        <developerConnection>scm:git:ssh://github.com/NVIDIA/spark-xgboost.git</developerConnection>
-        <url>https://github.com/NVIDIA/spark-xgboost</url>
+        <connection>scm:git:git:/github.com/rapidsai/xgboost.git</connection>
```
Comment (wbo4958) [inline jvm-packages/pom.xml]: Seems we should change to NVIDIA/spark-xgboost

File: jvm-packages/xgboost4j-example/pom.xml [hunk 2]
```diff
@@ -4,12 +4,12 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>ai.rapids</groupId>
+        <groupId>com.nvidia</groupId>
 	<artifactId>xgboost-jvm_2.11</artifactId>
```
Comment (wbo4958) [inline jvm-packages/xgboost4j-example/pom.xml:8]: should we change it to 2.12?

File: tests/cpp/common/test_device_helpers.cu [hunk 3]
```diff
@@ -96,84 +96,6 @@ TEST(bulkAllocator, Test) {
   TestAllocator();
 }
 
-template <typename T, typename Comp = thrust::less<T>>
```
Comment (wbo4958) [inline tests/cpp/common/test_device_helpers.cu:99]: have we touched this test?

File: jvm-packages/xgboost4j-example/pom.xml (hunk 2 above)
Comment (firestarman) [reply jvm-packages/xgboost4j-example/pom.xml:8]: example is not used

File: tests/cpp/common/test_device_helpers.cu (hunk 3 above)
Comment (firestarman) [reply tests/cpp/common/test_device_helpers.cu:99]: I do not know, but it shoud keep the same with our internal branch

File: jvm-packages/pom.xml (hunk 1 above)
Comment (firestarman) [reply jvm-packages/pom.xml]: nice catch, updated

File: jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/rapids/GpuColumnVectorUtils.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/*
+ Copyright (c) 2014 by Contributors
```
Comment (wbo4958) [inline jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/rapids/GpuColumnVectorUtils.java]: how about the old file of ai/rapids ?

Comment (firestarman) [reply jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/rapids/GpuColumnVectorUtils.java]: removed, nice catch

**Review by wbo4958 — APPROVED** [review]

**Pull request merged**